vignettes/Introduction.Rmd

```diff
@@ -180,13 +178,10 @@ n$x
 
 If the function takes no arguments, it's not possible to use it with `<-`:
 
-```{r eval=FALSE}
+```{r error=TRUE}
```

Comment:
The reason why it has `eval=FALSE` and the results are hard-coded into the text is because otherwise, the document builds with a different result each time. So it should stay the way it was.

Reply:
I only see one code chunk which makes the document build stochastic. 

````
```{r eval=FALSE}
n$rand
n$rand
n$rand <- 3
```
````

But this can be made reproducible by setting a seed:

````
```{r error=TRUE}
set.seed(123) # for reproducibility

n$rand
n$rand
n$rand <- 3
```
````

Is that an acceptable solution for you?

For the rest of the chunks that had `eval=FALSE`, the output is always the same.

Reply:
I don't think adding the `set.seed()` is an improvement over the original version, because it adds more noise for the reader. I suppose you could put it in a block just above with `echo=FALSE`.